base/cvd/cuttlefish/host/libs/web/http_client/http_client.cc

```diff
@@ -172,12 +172,19 @@ class CurlClient : public HttpClient {
   Result<HttpResponse<std::string>> DownloadToFile(
       const std::string& url, const std::string& path,
       const std::vector<std::string>& headers) override {
-    LOG(DEBUG) << "Attempting to save \"" << url << "\" to \"" << path << "\"";
+    LOG(DEBUG) << "Saving '" << url << "' to '" << path << "'";
 
     auto [shared_fd, temp_path] = CF_EXPECT(SharedFD::Mkostemp(path));
     SharedFDOstream stream(shared_fd);
-    auto callback = [&stream](char* data, size_t size) -> bool {
-      LOG(DEBUG) << "Downloaded chunk of " << size << " bytes";
+    uint64_t total_dl = 0;
+    uint64_t last_log = 0;
+    auto callback = [&stream, &total_dl, &last_log](char* data,
+                                                    size_t size) -> bool {
+      total_dl += size;
+      if (total_dl / 2 >= last_log) {
+        LOG(DEBUG) << "Downloaded " << total_dl << " bytes";
+        last_log = total_dl;
+      }
```

Comment:
It looks like this doesn't print the total size of the downloaded file at the end, is that possible?

Reply:
Done